article/app/services/dotcomponents/DotcomponentsLogger.scala

```diff
@@ -76,6 +76,12 @@ case class DotcomponentsLogger(request: Option[RequestHeader]) extends GuLogging
     )
   }
 
+  def logRequest(msg: String, results: Map[String, String], page: PageWithStoryPackage)(implicit
```

Comment:
this is great, you could use this in the `articlepicker` rather than reimplementing it there - might be for a future PR though - up to you